src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -24,6 +23,9 @@ def test_no_space_cont_create(self):
         # probably should be -1007, revisit later
         err2 = "-1009"
```

Comment:
So -1009 is still needed?

Reply:
Not needed anymore. Removed from the code.